examples/README.md

```diff
@@ -1991,6 +1991,30 @@ To avoid these issues, the `ghpc_stage` function can be used to copy a file (or
 The `ghpc_stage` function will always look first in the path specified in the blueprint. If the file is not found at this path then `ghpc_stage` will look for the staged file in the deployment folder, if a deployment folder exists.
 This means that you can redeploy a blueprint (`gcluster deploy <blueprint> -w`) so long as you have the deployment folder from the original deployment, even if locally referenced files are not available.
 
+### Reservation for Scheduled Maintenance
+
+A maintenance event is when a compute engine stops a VM to perform a hardware or
+software update which is determined by the host maintenance policy. This can
+also affect the running jobs if the maintenance kicks in. Now, Customers can control scheduled maintenance
+of the cluster using the cluster toolkit. You can enable reservation for
+scheduled maintenance for your compute nodeset and slurm will reserve your node
+for maintenance during the maintenance window. If you try to schedule any jobs
+which overlap with the maintenance reservation, slurm would not schedule any
```

Comment:
Please add section (one sentence + link to Slurm doc) about adding `-t` to the `srun` to be able to allocate jobs to node with reservation created.